starter-code/src/functions-and-arrays.js

```diff
@@ -47,6 +97,18 @@ var wordsUnique = [
   'simple',
   'bring'
 ];
+function uniquifyArray(wordsUnique)
+{
+  for (var i=0;i<wordsUnique.length;i++) {
+    for (var j=i+1;j<wordsUnique.length;j++) {
+        if (wordsUnique[i]==wordsUnique[j]){ 
+            wordsUnique.splice(j,1);
```

Comment:
This is dangerous since `splice` method change the `length` of the array.  So can have problems in the loop.
If you are curious you can ask us about this.